locales/en-US/experiments.ftl

```diff
@@ -60,6 +60,29 @@ containersContributors6Title = Front End Security
 containersContributors7Title = Firefox UX
 containersContributors8Title = Firefox UX
 containersContributors9Title = Firefox UX
+firefoxlockboxDescription = Take your passwords everywhere with Firefox Lockbox, an iOS mobile app.
+firefoxlockboxIntroduction = <p>Firefox Lockbox lets you easily access your passwords everywhere. Logins are stored and synced securely, using 256-bit encryption, keeping your information safe. Easily copy username and password details to get into apps and websites. Never again find yourself locked out of your account while on your phone!</p> <p><strong>Firefox Lockbox requires a Firefox Account in order to access your previously saved logins. If you do not have an account set up, get started <a href="https://lockbox.firefox.com/faq.html#how-do-i-enable-sync-on-firefox">here</a>.</strong></p>
+firefoxlockboxDetails0Copy = Access username and password information for all the accounts you save to Firefox.
+firefoxlockboxDetails1Copy = Manage accounts in Firefox and sync changes automatically to Firefox Lockbox.
+firefoxlockboxDetails2Copy = Sort and filter on your accounts to quickly locate the one you need.
+firefoxlockboxContributors0Title = Sr. Product Manager
+firefoxlockboxContributors1Title = Engineering Manager
+firefoxlockboxContributors2Title = Sr. Interaction Designer
+firefoxlockboxContributors3Title = Engineering Architect
+firefoxlockboxContributors4Title = Mobile Engineer
+firefoxlockboxContributors5Title = Sr. Engineer
+firefoxlockboxContributors6Title = Mobile Engineer
+firefoxlockboxContributors7Title = Product Data Scientist
+firefoxlockboxContributors8Title = Sr. UX Researcher
+firefoxlockboxContributors9Title = Content Strategy Lead
+firefoxlockboxContributors10Title = Sr. UX Designer
+firefoxlockboxContributors11Title = QA Developer
```

Comment:
If you ✂️me from the `contributors:` list above (and please do), you'll also need to make sure you tweak these values, otherwise all the names+titles will get out of sync (unless you replace me with MBrandt -- although I'm guessing he has a much more glamorous job title than I).